src/main/java/com/flightstats/hub/dao/aws/S3WriteQueue.java

```diff
@@ -47,24 +44,30 @@
     @Inject
     private StatsdReporter statsdReporter;
 
+    S3WriteQueue(ContentDao spokeWriteContentDao, ContentDao s3SingleContentDao, StatsdReporter statsdReporter) {
```

Comment:
I think you might be able to annotate this with `@Inject` and add some `@Named` annotations to the content daos. Then you could make the dependencies `private final`...

Reply:
that was done in the latest commit